monai/transforms/transforms.py

```diff
@@ -441,10 +441,51 @@ def __call__(self, img):
         return data
 
 
-# if __name__ == "__main__":
-#     img = np.array((1, 2, 3, 4)).reshape((1, 2, 2))
-#     rotator = RandRotate90(prob=0.0, max_k=3, axes=(1, 2))
-#     # rotator.set_random_state(1234)
-#     img_result = rotator(img)
-#     print(type(img))
-#     print(img_result)
+@export
+class RandomZoom(Randomizable):
+    """Randomly zoomes input arrays with given probability within given zoom range.
+
+    Args:
+        prob (float): Probability of zooming.
+        min_zoom (float or sequence): Min zoom factor. Can be float or sequence same size as image.
+        max_zoom (float or sequence): Max zoom factor. Can be float or sequence same size as image.
+        order (int): order of interpolation. Default=3.
+        mode (str): Determines how input is extended beyond boundaries. Default is 'constant'.
```

Comment:
Suggest to list all the possible mode names in doc-string, otherwise, users need to check scipy document.
Thanks.

Reply:
Fixed.